pkg/flow/internal/controller/loader.go

```diff
@@ -586,67 +590,81 @@ func (l *Loader) EvaluateDependencies(c *ComponentNode) {
 	defer span.End()
 
 	logger := log.With(l.log, "trace_id", span.SpanContext().TraceID())
-	level.Info(logger).Log("msg", "starting partial graph evaluation")
+	level.Info(logger).Log("msg", "starting partial graph evaluation", "originator", c.NodeID())
```

Comment:
This log line probably needs to change, since this is now just queueing nodes for evaluation really.